.babelrc

```diff
@@ -1,3 +1,3 @@
 {
-  "presets": ["env", "react"]
-}
\ No newline at end of file
+  "presets": ["env", "react", "stage-0"]
```

Comment:
I guess your previous problem could be solve by just claiming the `isActive` function as class function like: 
```
isActive(id) {
    return this.props.activeTab === id;
  }
```
Let's not set `stage-0`  for babel precompiling, because this is not for standard syntax: https://tc39.github.io/process-document/